### PR TITLE
fix: fix lighthouse errors

### PR DESCRIPTION
**Pull request by hugomrdias**
- missing meta viewport
- missing preconnect for analytics
- lazy load images
- add sizes to images to avoid reflow and improve Cumulative Layout Shift score
- reduce the all images sizes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "lint-staged": "^10.2.6",
     "markdown-it-deflist": "^2.0.3",
     "markdown-it-footnote": "^3.0.2",
+    "markdown-it-image-lazy-loading": "^1.0.2",
+    "markdown-it-imsize": "^2.0.1",
```
Comment (johnnymatthews) [inline package.json:18]: Does `markdown-it-imsize` also _optimize_ images?

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -20,7 +20,9 @@ module.exports = {
       md.use(require('markdown-it-video'))
       md.use(require('markdown-it-footnote'))
       md.use(require('markdown-it-task-lists'))
-      md.use(require('markdown-it-deflist'))
+      md.use(require('markdown-it-deflist')),
+        md.use(require('markdown-it-imsize')),
+        md.use(require('markdown-it-image-lazy-loading'))
```
Comment (cwaring) [inline docs/.vuepress/config.js:25]: Indentation looks a bit funky here, prettier should have fixed this? 🤔

File: docs/.vuepress/head.js [hunk 3]
```diff
@@ -1,4 +1,9 @@
 module.exports = [
+  [
+    'meta',
+    { name: 'viewport', content: 'width=device-width, initial-scale=1' }
+  ],
```
Comment (cwaring) [inline docs/.vuepress/head.js:5]:
```suggestion
```

This is already included in the [default SSR template](https://github.com/vuejs/vuepress/blob/master/packages/%40vuepress/core/lib/client/index.ssr.html).

File: docs/.vuepress/head.js [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+  ['link', { rel: 'preconnect', href: 'https://www.google-analytics.com' }],
```
Comment (cwaring) [inline docs/.vuepress/head.js:6]: Do we really want to preconnect if this resource is loaded async anyway and already non-blocking?

File: docs/how-to/command-line-quick-start.md [hunk 5]
```diff
@@ -16,9 +16,9 @@ Don’t want to use the command line right now? Give the desktop-app implementat
 
 Installing IPFS is simple, but varies between operating system:
 
-| [Windows](#windows) | [macOS](#macos) | [Linux](#linux) |
-| --- | --- | --- |
-| [![Windows icon](./images/command-line-quick-start/windows-icon.png)](#windows) | [![macOS icon](./images/command-line-quick-start/apple-icon.png)](#macos) | [![Linux icon](./images/command-line-quick-start/linux-icon.png)](#linux) |
+| [Windows](#windows)                                                                                 | [macOS](#macos)                                                                               | [Linux](#linux)                                                                               |
+| --------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
```
Comment (cwaring) [inline docs/how-to/command-line-quick-start.md:20]: Do you have a markdown formatter that's introducing this spacing? The previous spacing was easier to read in raw format.

File: package.json (hunk 1 above)
Comment (hugomrdias) [reply package.json:18]: no, just allows for width and height props

Comment (cwaring) [reply package.json:18]: Nope, it just allows you to set default display sizes. Which helps page load time and helps lower [CLS](https://web.dev/cls/).

File: docs/how-to/command-line-quick-start.md (hunk 5 above)
Comment (hugomrdias) [reply docs/how-to/command-line-quick-start.md:20]: the previous formatting was made by me manually something is adding this 

```
"lint-staged": {
    "*.{js,json,css,less,scss,html,md,vue}": [
      "prettier --write"
    ],
    "*.styl": [
      "stylus-supremacy format --replace"
    ]
  },
```

File: docs/.vuepress/head.js (hunk 4 above)
Comment (hugomrdias) [reply docs/.vuepress/head.js:6]: this come from lighthouse verbatim

Comment (cwaring) [reply docs/.vuepress/head.js:2]: ok then, I guess google loves been connected to everything ⚡

**Pull request merged**